core/templates/pages/exploration-player-page/learner-experience/continue-button.component.ts

```diff
@@ -13,20 +13,24 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the Continue button in exploration player and
+ * @fileoverview Component for the Continue button in exploration player and
  * editor.
  */
 
-angular.module('oppia').directive('continueButton', [
-  function() {
-    return {
-      restrict: 'E',
-      scope: {
-        onClickContinueButton: '&',
-        isLearnAgainButton: '&',
-        focusLabel: '@'
-      },
-      template: require('./continue-button.directive.html')
-    };
-  }
-]);
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+@Component({
+  selector: 'continue-button',
+  templateUrl: './continue-button.component.html',
+  styleUrls: []
+})
+export class ContinueButtonComponent {
+  @Input() focusLabel: string;
+  @Input() isLearnAgainButton: boolean = false;
```

Comment:
Earlier `isLearnAgainButton` was a function, right? (If we are changing it to boolean then change the variable name as the variable name should not start with a verb.)

Also, can you please test the changes for the "Learn again" button?

Reply:
Where can I find the "Learn again" button? I tried publishing an exploration and I went through the exploration, but there was no "Learn again" button at the end.